react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
https://github.com/burge-is/react-live/blob/24be04a17906ff8641d1df29cc0f4c00dce24717/react-live.css#L13
^ this line make the Editor have no outline.  The TabGate code from before had a wrapper so this piece of CSS wasn't active.